deeprank_gnn/NeuralNet.py

```diff
@@ -293,39 +342,41 @@ def train(self, nepoch=1, validate=False, save_model='last',
 
                 train_losses.append(loss_)
 
-
                 # Validate the model
                 if validate:
 
-                    t0 = time()
+                    # t0 = time()
```

Comment:
time is not actually used. Just confirm that I can remove it

Reply:
Yes, let's remove it.